packages/gitgraph-react/src/Gitgraph.tsx

```diff
@@ -112,21 +108,20 @@ class Gitgraph extends React.Component<GitgraphProps, GitgraphState> {
           {this.renderBranchesPaths()}
           <g ref={this.$commits}>
             {this.state.commits.map((commit) =>
-              <CommitComp
+              <Commit
                 key={commit.hashAbbrev}
                 commits={this.state.commits}
                 commit={commit}
                 currentCommitOver={this.state.currentCommitOver}
                 setCurrentCommitOver={this.setCurrentCommitOver.bind(this)}
                 gitgraph={this.gitgraph}
-                initCommitElements={this.initCommitElements.bind(this)}
-                commitsElements={this.commitsElements}
                 getWithCommitOffset={this.getWithCommitOffset.bind(this)}
                 setTooltip={this.setTooltip.bind(this)}
+                commitMessagesX={this.state.commitMessagesX}
               />
             )}
           </g>
-          {this.$tooltip}
+          {this.$tooltip}[
```

Comment:
❗ This looks like a typo, nope?

```suggestion
          {this.$tooltip}
```